src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/_params.py

```diff
@@ -33,16 +33,19 @@ def validate_locations(ns):
         loc_dict.append(Location(location_name=comps[0], failover_priority=int(comps[1])))
     ns.locations = loc_dict
 
+def validate_ip_range_filter(ns):
+    ns.ip_range_filter = ",".join(ns.ip_range_filter.split())
```

Comment:
If the range filter isn't specified, this will crash.
```Python
def validate_ip_range_filter(ns):
  if ns.ip_range_filter:
    ns.ip_range_filter = ','.join(ns.ip_range_filter)
```

With nargs=+ specified in the parameter registration, this property will already contain a list of strings, so no `split` is necessary. Using split would require a space separate list of filters all surrounded by quotes ("filter1 filter2 ..."), which is also not a pattern we do not permit.